internal/services/monitor/monitor_workspace_resource_test.go

```diff
@@ -111,6 +111,27 @@ func TestMonitorWorkspace_publicNetworkAccess(t *testing.T) {
 	})
 }
 
+func TestMonitorWorkspace_privateEndpointConnection(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_monitor_workspace", "test")
+	r := WorkspaceTestResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.privateEndpointConnection(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
```

Comment:
can we check the properties from the `private_endpoint_connections` block have values here?